DataFormats/PatCandidates/src/PackedCandidate.cc

```diff
@@ -335,6 +335,17 @@ float pat::PackedCandidate::puppiWeight() const { return unpack8logClosed(packed
 
 float pat::PackedCandidate::puppiWeightNoLep() const { return unpack8logClosed(packedPuppiweightNoLepDiff_+packedPuppiweight_,-2,0,64)/2. + 0.5;}
 
+void pat::PackedCandidate::setRawCaloFraction(float p) {
+  if(p>1)
+    rawCaloFraction_ = 250; // Set to overflow value of 250 (since uint8 only goes up to 256)
```

Comment:
can this be a ``` std::numeric_limits<uint8_t>::max()```
250 is not particularly obvious overflow value.

Also, given that there is nothing particularly special about this fraction value be slightly above 1, this if should probably be less strict and allow values above 1 saved up to a larger value; the format supports values up to 2.54